src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -50,6 +50,15 @@ class UniversalNavbar extends React.Component {
   }
 
   render() {
+    const Link = ({ href, ...props }) => {
```

Comment:
What happens if I wanna use Link somewhere else? I can't. Is there any reason to not make Link a sharable component of it's own?

Reply:
This `Link` is created from the props from UniversalNavbar. I guess we can create a Link HOC component/function.